gettsim/benefits/wohngeld.py

```diff
@@ -264,3 +264,33 @@ def apply_wg_formula(household, params, household_size):
         )
 
     return wg_amount
+
+
+def regrouped_wohngeld_formel(household, params, household_size):
+    # The formula is only valid for up to 12 household members
+    household_size_max = min(household_size, 12)
+    koeffizenten = params["koeffizienten"][household_size_max]
```

Comment:
Just use `min(household_size, 12) ` directly, the max-version is never used again and makes it harder to read IMHO.